lib/platform/gitlab/index.js

```diff
@@ -50,14 +51,11 @@ module.exports = {
 async function getRepos(token, endpoint) {
```

Comment:
Does this syntax still make sense? i.e. passing token and endpoint to this function?